packages/foam-vscode/src/features/tags-tree-view.spec.ts

```diff
@@ -94,7 +95,9 @@ describe('Tags tree panel', () => {
 
     childTreeItems.forEach(child => {
       if (child instanceof TagItem) {
+        // eslint-disable-next-line jest/no-conditional-expect
```

Comment:
I see why this rule is helpful, if we changed the instance of `child` this test wouldn't fail, but would silently skip any check.. maybe we should actually remove the `instanceof` and find a way to incorporate that in the test

Reply:
Yes, I'd like to change it, but I'm trying to avoid non-trivial refactors in this PR. That's why I've disabled some failing checks. I think it's an overall gain anyway: in the future we will be better advised not to do some things, while we still have a mark (we can search for eslint-disable) for previous violations of the rule. Maybe I can locally make this a warning instead of disabling it, I'm not sure if that's possible.

Reply:
> Maybe I can locally make this a warning instead of disabling it, I'm not sure if that's possible.

It's only possible using a cumbersome syntax that will require to first turn the rule into a warning and then, in another line, revert that setting. I think it's better to keep the oneliner.